gallery/gallery/tool/grind.dart

```diff
@@ -0,0 +1,81 @@
+import 'dart:io';
```

Comment:
I would suggest moving this file up outside the gallery directory

Reply:
Done.

Reply:
Actually after doing this I realized that this will make Android Studio confused when you open the gallery directory. 

It will then not recognize that the sub directories also have pubspec.yaml files and you have to open each directory separately. Not sure if it's worth moving this down one directory as its probably adds more confusion than help.